bounce/.pylintrc

```diff
@@ -0,0 +1,546 @@
+[MASTER]
```

Comment:
Can you remove this file please? Since it's a personal text editor configuration file we should keep it out of the repo.